+bisos/+iteration/+step/termination.m

```diff
@@ -0,0 +1,111 @@
+classdef termination < bisos.iteration.Step
+   
+properties
+    type = 'termination';
+    varout = [];
+    varin;
+    data_prev = true;
+    
+    % save function handles and variables in play (maybe a struct would be nice)
+    operator;
+    
+    fhan1; 
+    vars1; % previous variables
+    
+    fhan2;
+    vars2; % current variables
+    
+    
+    
+end
+
+methods
+    function step = termination(~, previous, current, op, varargin)
+        % setup termination rule
+        
+        % get variabled needed as input in a certain cycle and save
+        % function handles and other variables needed
+        if length(current)==1
+            % in case NO function handle is defined
+            for i = 1:length(current)
+                step.varin{length(step.varin)+1}=current{i};
+            end
+            step.vars2 = current;
+        else
+            % in case some function handle is defined
+            for i = 1:length(current{2})
+                step.varin{length(step.varin)+1}=current{2}{i};
+            end
+            step.vars2 = current{2};
+            step.fhan2 = current{1};
+        end
+        
+        if length(previous)==1
+            % in case NO function handle is defined
+            step.vars1 = previous;
```

Comment:
what about `step.fhan1` here?

Reply:
The idea here is in case no fhandle is given, no assignment is made to it.